### ISSUE TITLE
AA amyloidosis

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6118]
**Preferred term label:**
AA amyloidosis

**Synonyms**
secondary amyloidosis

**Definition (free text, please give PubMed ID)**
Extracellular tissue deposition of fibrils that are composed of fragments of and/or intact serum amyloid A protein (SAA), a hepatic acute phase reactant. 29261990

**Parent term (use hpo.jax.org/app)**
HP:0011034

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
105210

**Your nano-attribution (ORCID)**
0000-0003-3411-9598

**Comment by pnrobinson** [comment]
OMIM:142680 
PMID: 23965844   AA amyloidosis has developed in 16 (10%)patients at a median age of 43 years.  16/158

**Comment by pnrobinson** [comment]
adding. In the future we may want to subdivide amyloidosis as by location and by protein
